### PR TITLE
Feature/cypress adjustment home page

### PR DESCRIPTION
**Pull request by joanafbrito**
# Description drawing

-Worked Refactoring `home_page_spec.js` to match the changes in our project

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactor feature (non-breaking change that modifies existing work)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] local host 3000
- [ ] console.log()
- [x] dev tools
- [x] cypress testing

# Checklist:

- [x] My code follows the style guidelines of this project/linted
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: cypress/integration/home_page_spec.js [hunk 1]
```diff
@@ -4,32 +4,33 @@ describe('Home page user flow', () => {
         cy.onLoad()
     });
 // 
-    it('Should contain a Header (the Navigation bar) with a How to and a Image ', () => {
+    it('Should contain a Header with a How to and the Images ', () => {
         cy.get('.header')
           .should('be.visible')
-          .get('.header-image')
+          .get('.ash')
```
Comment (hayleyw7) [inline cypress/integration/home_page_spec.js:10]: Great thinking testing for the images too.

File: cypress/integration/home_page_spec.js [hunk 2]
```diff
@@ -4,32 +4,33 @@ describe('Home page user flow', () => {
...
-        //   we need to add test for the animations
     });
 
     it('Should be able to click on How to and go to the How To page', () => {
         cy.get('a')
           .click()
           .get('.all-pokemon')
           .should('not.exist')
-          .get('.help-page')
+          .get('.howto-page')
```
Comment (hayleyw7) [inline cypress/integration/home_page_spec.js:24]: Thanks for changing all of the `help`s to `howto`s.

**Review by hayleyw7 — APPROVED** [review]
Looks great. Thanks again!

File: src/components/Header/Header.js [hunk 3]
```diff
@@ -4,26 +4,27 @@ import { Route, Link } from 'react-router-dom';
...
+  //   },
+  //   {   
+  //       scale: 1,
+  //       opacity: 0,
+  //       duration: 2.5, 
+  //       ease: "rough({ template: none.out, strength: 1, points: 20, taper: 'none', randomize: true, clamp: false})", 
+  //       y: 0,   
+  //       rotation: 0
+  //   })
+  // }, [ foundPokemon]);
```
Comment (Matt-Roden) [inline src/components/Header/Header.js:27]: Still awesome!

**Review by Matt-Roden — APPROVED** [review]

**Pull request merged**